src/wallet.rs

```diff
@@ -4,6 +4,15 @@ use anyhow::Result;
 use reqwest::Url;
 use serde::{Deserialize, Serialize};
 
+// TODO: Either use println! directly or import tracing also?
+use std::println as debug;
+
+// TODO: Consider using bignum for moneroj instead of u64?
```

Comment:
FYI: Bignum is not copy which can be annoying.

Reply:
Noted, cheers.